oxidation/libparsec/crates/client_types/Cargo.toml

```diff
@@ -31,7 +31,12 @@ paste = "1.0.9"
 regex = "1.6.0"
 thiserror = "1.0.37"
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
```

Comment:
If we need this, it is a strong indication there should be a new platform crate to store this stuff (`platform_device_loader` that will contain the fs access, access to the to-be-developped server-side storage of device file, and the mock of fs access for web)

Reply:
What's the difference with `platform_native` and `platform_web` then ?

Reply:
Those crate should be removed

Reply:
cf. https://github.com/Scille/parsec-cloud/pull/3908

Reply:
I don't understand, do you want I recreate something like these old platform ?